hrjobcontract/js/test/controllers/contract_test.js

```diff
@@ -1,22 +1,57 @@
 define([
+    'common/moment',
     'common/angularMocks',
-    'job-contract/app'
+    'job-contract/app',
+    'job-contract/controllers/contract'
 ], function () {
     'use strict';
 
     describe('ContractCtrl', function () {
-        var ctrl;
+        var ctrl, $rootScope, $scope, $controller;
 
         beforeEach(module('hrjc'));
-        beforeEach(inject(function ($controller, $rootScope) {
-            var $scope = $rootScope.$new();
-            $scope.contract = { id: '1' };
+        beforeEach(inject(function (_$controller_, _$rootScope_) {
+
+          $controller = _$controller_;
+          $rootScope = _$rootScope_;
+
+          $scope = $rootScope.$new();
+
+          $scope.contract = {
+            id: '1',
+            contact_id: "84",
+            deleted: "0",
+            is_current: "0",
+            is_primary: "1"
+          };
+
+          $scope.$parent.contract = {
+            id: '1',
+            contact_id: "84",
+            deleted: "0",
+            is_current: "0",
+            is_primary: "1"
+          };
+
+          $scope.$parent.contractCurrent = [];
+          $scope.$parent.contractPast = [];
```

Comment:
Can't you encapsulate all this code here
```js
$scope.contract = {
  id: '1',
  contact_id: "84",
  deleted: "0",
  is_current: "0",
  is_primary: "1"
};

$scope.$parent.contract = {
  id: '1',
  contact_id: "84",
  deleted: "0",
  is_current: "0",
  is_primary: "1"
};

$scope.$parent.contractCurrent = [];
$scope.$parent.contractPast = [];
```
in a function? Something like this:
```js
beforeEach(inject(function (_$controller_, _$rootScope_) {
  $controller = _$controller_;
  $rootScope = _$rootScope_;
  $scope = $rootScope.$new(); 

  setupData(); // <-- temporary name

  ctrl = $controller('ContractCtrl', { $scope: $scope });
}));
```
it would make the code more readable